src/deltastats.ts

```diff
@@ -73,6 +73,20 @@ export function startDeltaStatistics(
   return setInterval(() => {
     updateProviderPeriodStats(app)
     const anyApp = app as any
+    const config = getSecurityConfig(anyApp)
+    const showDeviceLabelNames = config?.showDeviceLabelNames !== false
+    let devices: any[] = []
+    if (
+      anyApp &&
+      anyApp.securityStrategy &&
+      typeof anyApp.securityStrategy.getDevices === 'function'
+    ) {
+      devices = anyApp.securityStrategy.getDevices(config)
+      // Strip descriptions when toggle is OFF so UI shows IDs
```

Comment:
We don't want to expose any parts of our security config to all, also unauthenticated users like this, so that optionally a property is stripped.

Instead only the properties that we deem necessary are sent over the ws.

See https://github.com/SignalK/signalk-server/blob/5a130431805bdac72a464c2da504f9f0c0d68fc7/src/security.ts#L83-L89

There's also a "future problem": if all properties are sent over ws a future developer may add something more sensitive to Device and it will get exposed.

So instead just pick the properties absolutely needed.